habitat-baselines/habitat_baselines/rl/hrl/hierarchical_policy.py

```diff
@@ -223,7 +225,10 @@ def _broadcast_skill_ids(
         for k, v in skill_to_batch.items():
             grouped_skills[k] = (
                 v,
-                {dat_k: dat[v] for dat_k, dat in sel_dat.items()},
+                {
+                    dat_k: dat if dat is None else dat[v]
```

Comment:
Are we okay with having dat be Any ? In which cases can this be null ? 
```suggestion
                    dat_k: None if dat is None else dat[v]
```

Reply:
Actually it's okay to leave how it was before. Thanks for pointing this out.